.ci.yaml

```diff
@@ -11,6 +11,23 @@ enabled_branches:
   - flutter-\d+\.\d+-candidate\.\d+
 
 platform_properties:
+  ubuntu:
```

Comment:
Is this the most elegant way of doing the switchover, or is there another way to perform a staggered switch of builds over to the new Ubuntu hosts.

Reply:
I think the most elegant way would be switching the os over in Linux so you don't need to duplicate builders for the migration.

If there's specific targets failing, or we don't have capacity to do it all at once, we could set os in specific targets.

Reply:
I would like to do the switch over in a staggered manner. So if we can override the os: property in each builder, maybe thats the right way to do it and then switch over the linux: property group and take out the overrides?

Reply:
@godofredoc what do you think?

Reply:
Could we enable all linux vm tests in staging to run against the ubuntu bots to validate first, see for a couple of days? Then we are pretty confident to switch over to `os: ubuntu`, and inject `os: linux` for targets that are failing.